salt/elasticsearch/plugins/readonlyrest.sls

```diff
@@ -12,45 +12,27 @@ readonlyrest-install:
     - watch_in:
       - service: elasticsearch
 
-/opt/pkcs-password:
+/opt/restart-elasticsearch.sh:
   file.managed:
-    - name: /opt/pkcs-password
-    - contents_pillar: elasticsearch:plugins:readonlyrest:key_pass
-    - mode: 600
-
-/opt/pem-to-keystore.sh:
-  pkg.installed:
-    - name: openjdk-11-jre-headless # for keytool command
-  file.managed:
-    - name: /opt/pem-to-keystore.sh
-    - source: salt://elasticsearch/files/pem-to-keystore.sh
-    - mode: 700
-    - require:
-      - pkg: apache2
-      - pkg: elasticsearch
-      - pkg: /opt/pem-to-keystore.sh
-      - file: /opt/pkcs-password
-
-/opt/pem-to-keystore-wrapper.sh:
-  file.managed:
-    - name: /opt/pem-to-keystore-wrapper.sh
-    - source: salt://elasticsearch/files/pem-to-keystore-wrapper.sh
+    - name: /opt/restart-elasticsearch.sh
+    - source: salt://elasticsearch/files/restart-elasticsearch.sh
     - user: root
     - group: root
     - mode: 755
```

Comment:
Change to 750 if only run via sudo/root?

Reply:
Done ✔

Reply:
This needs to be executable (and not editable) for the ``www-data`` user which 750 will block.

Alternatively we could set the group to ``www-data`` however this is quite specific to the docs server and won't scale as easily if other users need access.

Reply:
Ah, right, this file is run by www-data, and the command within this file is sudo 👍